README.md

```diff
@@ -5,76 +5,78 @@
 
-### Steps ###
-* Navigate to the page chrome://extensions
-* Toggle the Developer mode in the top right corner
-* Click Load Unpacked Extension...
-* Select the [Lightning Inspector Directory]/builds/[version||latest]
 
+## Installs and Set-up  
 
-### Environment
+1. Download or clone the Lightning Inspector code from Github and unzip the file.
+2. Set up the Environment
 ```sh
-nvm install 5.0.0       # >= 5.0.0 is ok (don't install anything with sudo)
-npm install yarn -g     # install yarn package manager
+  1. nvm install --lts
+  2. npm install yarn -g 
 ```
-
-### Setup
+3. Set up
 ```sh
-yarn install            # install project dependencies (and submodules)
+  yarn install 
 ```
-### Development
+4. Development 
 ```sh
-yarn watch              # watches for changes in submodules
+  yarn watch 
 ```
 
-- **Plugin**: Open load the unpacked extension and changes should be reflected once `yarn watch` is issued
 
-### Distribution
-```sh
-yarn build              # should output crx, xpi, zip files
-```
+## Running the Inspector   
+
+1. Open up a browser like Google Chrome and type in chrome://extensions/ 
+2. Click the Developer Mode checkbox in the top right
+3. Click the "Load unpacked Extensions" button on the left 
+4. Select the directory for the aura-inspector
+5. On a website you wish to use the inspector, right click on page and click on 'inspect' (ctrl/cmd + option + i).
+6. From there, navigate on the top bar with tabs to access the Lightning Inspector tool
+
 
-## How to Use ##
-The google sites page and public documentation are still great resources.
-https://sites.google.com/a/salesforce.com/user-interface/aura/aura-dev-tools
 
+## How to Use 
+
+The google sites page and public documentation are still great resources.
 
+https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/inspector_use.htm
 
-## Contribute
-Fork the repo and send the pull request. 
 
-Please add Kris Gray as the reviewer.
 
+## PR Requirements and Testing Specifics 
 
-## Backlog
+1. When making changes to the Lightning Inspector code, there are a set of requirements that must pass before sending a PR. 
+2. First, fork the repository on Github. 
+3. If you wrote a unit test for the changes, your test and the current unit tests in the code base should all pass. You can do this by running yarn test on the terminal. 
+4. The inspector should build without any errors. You can do this by running ‘yarn build’ on the terminal. 
+5. The inspector should also pass with manual testing. 
+    1. Go to the chrome extension page, chrome://extensions/, and press the reload button for the Lightning Inspector extension. 
+    2. The inspector should work with Lightning Experience [one.app](http://one.app/), Lightning Out, and non aura pages should still show a message. Go through all the tabs under the Lightning Inspector, and if you see an error there needs to be a fix. 
```

Comment:
Wrong url for one.app

Reply:
If you see an error, verify against master. If there is still an error, file an issue to the repo and you may proceed with your PR.